sdcm/nemesis.py

```diff
@@ -846,6 +846,55 @@ def _add_drop_column_run_in_cycle(self):
             self._add_drop_column()
             time.sleep(1)
 
+    def choose_partitions_for_delete(self, partitions_amount, ks_cf, with_clustering_key_data=False):
+        # This parameter is maximum -partition-count that data is written with -validate-data
+        max_partitions_with_data_validate = self.cluster.params.get('max_partitions_with_data_validate', 0)
+
```

Comment:
parameter name isn't clear enough.
Maybe change to 'partition_range_with_data_validation': '0-1000'.

Reply:
Done

Reply:
done